src/reporter.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+import { EnvMissingError } from './validators'
+
+const colorWith = (colorCode: string) => (str: string) => `\x1b[${colorCode}m${str}\x1b[0m`
+const colors = {
+  blue: colorWith('34'),
+  white: colorWith('37'),
+  yellow: colorWith('33'),
+}
```

Comment:
Nice, TIL how to color things

Reply:
Yeah, it's ugly to read but it works! Just a _little bit_ smaller implementation than having that `chalk` dependency :P